rotkehlchen/db/dbhandler.py

```diff
@@ -1145,21 +1145,27 @@ def add_blockchain_accounts(
             account_data: list[BlockchainAccountData],
     ) -> None:
         # Insert the blockchain account addresses and labels to the DB
-        blockchain_accounts_query = []
-        address_book_query = []
+        blockchain_accounts_query, bindings_to_update, bindings_to_delete = [], [], []
```

Comment:
you don't need bindings_to_update/delete for this method as it is used to **only** add an account.

plus, you only add to addressbook iff a label is provided i.e it's not falsy.